packages/edit-navigation/src/components/layout/unselected-menu-state.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Card, CardBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AddMenu from '../add-menu';
+import MenuSwitcher from '../menu-switcher';
+
+export default function UnselectedMenuState( {
+	onCreate,
+	onSelectMenu,
+	menus,
+} ) {
+	const showMenuSwitcher = menus?.length > 0;
+	return (
+		<div className="edit-navigation-empty-state">
+			{ showMenuSwitcher && <h4>{ __( 'Choose a menu to edit:' ) }</h4> }
+			<Card>
+				<CardBody>
+					{ showMenuSwitcher ? (
+						<MenuSwitcher
```

Comment:
I think `MenuSwitcher` does still need to be wrapped with a `NavigableMenu`, that will.

It can't be added inside the `MenuSwitcher` component. `MenuSwitcher` is also used in `DropdownMenu` which adds `NavigableMenu` itself.